backend/server/routers/programs.py

```diff
@@ -158,11 +160,26 @@ def convertSubgroupObjectToCoursesDict(object: str, description: str|list[str]):
 
 def addSubgroupContainer(structure: dict, type: str, container: dict, exceptions: list[str]) -> list[str]:
     """ Returns the added courses """
-    structure[type][container["title"]] = {}
-    item = structure[type][container["title"]]
-
+    #TODO: further standardise non_spec_data to remove these line:
+    title = container.get("title")
+    if container.get("type") == "gened":
+        title = "General Education"
+    
+    # don't do anything if it's info_rule or limit_rule
+    if container.get("type") is not None and "rule" in container.get("type"):
+        return []
```

Comment:
We going to do anything in the future with it? Some of the limit rules are important, and some of the information rules just seem like something good to display to the user (they're just in plain English so there's no shot we'll format them)

Reply:
yup we don't have a place to display them yet but i agree we should. will ask the FE team to do that after this is merged